gulpfile.babel.js

```diff
@@ -1,4 +1,6 @@
 /* eslint-disable import/no-extraneous-dependencies*/
+require('babel-polyfill');
```

Comment:
I think it was because we didn't have anything that used generators. I added a generator in this pr, and since pages are rendered at build time, there was an error because regeneratorRuntime wasn't available. This fixes that.